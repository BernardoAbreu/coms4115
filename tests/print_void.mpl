include "../DaMPL_stdlib.mpl";

/* This function returns void */
fun print10Hellos() {
	for i in range(0, 10) {
		print("Hello\n");
	}
}

print( print10Hellos() );