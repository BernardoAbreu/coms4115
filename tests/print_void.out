Fatal error: exception Failure("builtin fun print does not accept such params")
